index.html

```diff
@@ -1216,6 +1216,29 @@ <h2>Bindings for Events</h2>
         </p>
       </section>
 
+
+
+    </section>
+    <section>
+      <h1>Guideline to write new Protocol Binding</h1>
+      <p>To provide a new binding specification, e.g. based on a new communication protocol, a 
+        minimal requirements for information are required, which can then be used for distinct 
+        implementation and to ensure interoperability based on the new binding.
+      <p>
+        <!--be associated with WoT's <a>Properties</a>,
+      <a>Actions</a>, and <a>Events</a> the following minimum of requirment of information is required:</p>
+        -->
+      <p>
+        Please ensure that following information are provided and common shared:
```

Comment:
*Please ensure that following information are provided and common shared* -> *Please ensure that the following information is provided and commonly shared*